carapace-server/src/main/java/org/carapaceproxy/core/RuntimeServerConfiguration.java

```diff
@@ -27,6 +27,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.sql.Timestamp;
 import java.text.SimpleDateFormat;
+import java.time.format.DateTimeFormatter;
```

Comment:
remove?

Reply:
You're right, I reverted modifications to this file, as they weren't useful anymore